ksql-common/src/main/java/io/confluent/ksql/schema/connect/SqlSchemaFormatter.java

```diff
@@ -126,7 +126,7 @@ protected String visitBytes(final Schema schema) {
           + DecimalUtil.scale(schema) + ")";
     }
 
-    throw new KsqlException("Cannot format bytes type: " + schema);
+    return "BYTES";
```

Comment:
It's still the case that the only bytes type we currently support is Decimals, right? I'm having trouble understanding why this change is necessary (besides the pretty error message in UdfIndexTest#shouldNotFindArbitraryBytesTypes`).

Reply:
just the pretty error message - if you can't find a UDF we shouldn't throw "Cannot format bytes type". The other BYTES type that we support is also the way we represent Generics.